spec/features/01_user_authentication_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+require 'launchy'
+
+feature 'authenticates a user' do
+  before do
+    user_1 = create(:user)
+    visit '/'
+  end
+
+  scenario 'user creates an account' do
+    click_link('Sign Up')
+
+    fill_in('First Name', :with => 'John')
+    fill_in('Last Name', :with => 'Doe')
+    fill_in('Enter password', :with => 'Seekrit')
+    fill_in('Confirm password', :with => 'Seekrit')
+    fill_in('Username', :with => 'john_doe123')
+    fill_in('Email', :with => 'johndoe@123.com')
+    fill_in('Profile Picture', :with => 'http://s3.amazonaws.com/37assets/svn/765-default-avatar.png')
+
+    click_link('Submit')
+
+    expect(page).to have_content("You have successfully signed in.")
+  end
+
+  scenario 'user signs in' do
+    click_link('Sign In')
+
+    fill_in('Username', :with => user_1.username)
+    fill_in('Password', :with => user_1.password)
+
+    click_link('Sign In')
+
+    expect(page).to have_content("You have successfully signed in.")
+  end
+
+  scenario 'user signs out' do
+
+    click_link('Sign In')
+
+    fill_in('Username', :with => user_1.username)
+    fill_in('Password', :with => user_1.password)
+
+    click_link('Sign In')
```

Comment:
You are repeating the sign in logic in all your tests. I'd recommend creating a helper authentication method called `sign_in_as` that takes in a user object.
https://github.com/davidrf/adventure_planner/blob/master/spec/support/helpers/authentication_helper.rb
Then include the `AuthenticationHelper` module in your `Rspec.config`
https://github.com/davidrf/adventure_planner/blob/master/spec/rails_helper.rb